content/docs/cosidriver/features/objectscale.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ObjectScale
+linktitle: ObjectScale
+weight: 1
+Description: Code features for ObjectScale COSI Driver
+---
+
+<!--
+TODO: what are the defaults with which bucket is provisioned? E.g. encryption, block size, and other? This needs to be described
+-->
+
+> **Notational Conventions**
+>
+> The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
+
+Fields are specified by theirs path. Consider the following examples:
+
+1. Field specified by the following path `spec.authenticationType=IAM` is reflected in their resources YAML as the following:
+
+```yaml
+spec:
+  authenticationType: IAM
+```
+
+2. field specified by path `spec.protocols=[Azure,GCS]` is reflected in their resources YAML as the following:
+
+```yaml
+spec:
+  protocols:
+    - Azure
+    - GCS
+```
+
+## Prerequisites
+
+In order to use COSI Driver on ObjectScale platform, the following components MUST be deployed to your cluster:
+
+- Kubernetes Container Object Storage Interface CRDs
+- Container Object Storage Interface Controller
+
+> ℹ️ **NOTE:** use [the official COSI guide](https://container-object-storage-interface.github.io/docs/deployment-guide#quick-start) to deploy the required components.
+
+## Kubernetes Objects
+
+### Bucket
+
+`Bucket` represents a Bucket or its equivalent in the storage backend. Generally, it should be created only in the brownfield provisioning scenario. The following is a sample manifest of `Bucket` resource:
+
+```yaml
+apiVersion: objectstorage.k8s.io/v1alpha1
+kind: Bucket
+metadata:
+  name: my-bucket
+spec:
+  driverName: cosi.dellemc.com
+  bucketClassName: my-bucket-class
+  bucketClaim: my-bucket-claim
+  deletionPolicy: Delete
+  protocols:
+    - S3
+  parameters:
+    id: "my.objectscale"
+```
+
+#### `spec.existingBucketID`
+
+`existingBucketID` is optional field that contains the unique id of the bucket in the ObjectScale. This field should be used to specify a bucket that has been created outside of COSI. Due to the fact, that the driver supports multiple arrays and multiple ObjectStores from one instance, the `existingBucketID` needs to have a format of: `<Configuration ID>-<Existing Bucket ID>`, e.g. `my.objectscale-existing-bucket`.
```

Comment:
Either:
is **an** optional field ...

-or remove the "is"-
optional field ....

Als0 - remove comma after "Due to the fact"